package/gocryptfs/package

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(gocryptfs)
+pkgdesc="An encrypted overlay filesystem written in Go."
+url="https://nuetzlich.net/gocryptfs/"
+_srcver=2.0-beta2
+pkgver="$_srcver"-1
+timestamp=2021-03-22
+section=utils
+maintainer="plan5 <30434574+plan5@users.noreply.github.com>"
+license=MIT
+
+depends=(fuse-utils)
+
+makedepends=(build:git)
```

Comment:
I'm guessing the build uses git?

Reply:
I'm checking if it's still necessary with the golang image

Reply:
It's not. I've removed it.